keyboards/momoka_ergo/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT(
+        KC_ESC,    KC_1,    KC_2,    KC_3,  KC_4,    KC_5, KC_6,    KC_7,    KC_8,    KC_9,  KC_0,    KC_MINS,
+        KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T, KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSLS,
+        KC_CAPS,    KC_A,    KC_S,    KC_D,    KC_F, KC_G, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+        KC_LSFT,    KC_Z,    KC_X, KC_C,  KC_V, KC_B, KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+        MO(_FN), KC_MINS,   KC_EQL, KC_LEFT, KC_RGHT, KC_LALT, KC_RGUI, KC_UP,   KC_DOWN, KC_LBRC, KC_RBRC, MO(_FN),
+        KC_BSPC, KC_LCTL, KC_HOME,  KC_PGUP, KC_RCTL, KC_SPC,
+        KC_DEL, KC_END,  KC_PGDN, KC_ENT
+    ),
+    [_FN] = LAYOUT(
+        RGB_TOG,    RGB_MODE_FORWARD,    RGB_MODE_RAINBOW,    KC_9,  KC_0,    KC_MINS, KC_6,    KC_7,    KC_8,    KC_9,  KC_0,    KC_MINS,
+        KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSLS, KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSLS,
+        KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+        KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+        KC_RGUI, KC_UP,   KC_DOWN, KC_LBRC, KC_RBRC, MO(_FN), KC_RGUI, KC_UP,   KC_DOWN, KC_LBRC, KC_RBRC, MO(_FN),
+        KC_BSPC, KC_LCTL, KC_HOME,  KC_PGUP, KC_RCTL, KC_SPC,
+        KC_DEL, KC_END,  KC_PGDN, KC_ENT
+    )
+};
+/*
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QMKBEST:
+            if (record->event.pressed) {
+                // when keycode QMKBEST is pressed
+                SEND_STRING("QMK is the best thing ever!");
+            } else {
+                // when keycode QMKBEST is released
+            }
+            break;
+        case QMKURL:
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+    }
+    return true;
+}
+*/
```

Comment:
```suggestion
```
I don't think it is very useful to include a big wall of commented-out code in a default keymap.